components/AnimationTiming/src/CAMediaTimingFunction+MDCAnimationTiming.h

```diff
@@ -28,37 +28,44 @@ typedef NS_ENUM(NSUInteger, MDCAnimationTimingFunction) {
    is slow both at the beginning and end. It has similar characteristics to the system's EaseInOut.
    This is known as Standard in the Material Design spec.
    */
-  MDCAnimationTimingFunctionEaseInOut,
```

Comment:
We'll need to keep these old API names around for some time while we migrate client teams over to the new APIs. Please add them back and alias them to the new API names.

Reply:
Should I switch lines 57-59?

`MDCAnimationTimingFunctionStandard = MDCAnimationTimingFunctionEaseInOut`

Reply:
Oh! I somehow missed that. That looks perfect.